pynsee/download/_get_value_label.py

```diff
@@ -42,7 +44,7 @@ def _add_metadata(id, df):
             df = pd.concat([df] + list_df_metadata, axis=1)
         
     else:
-        print("No column-specific metadata has been found for this file")
+        logger.info("No column-specific metadata has been found for this file")
```

Comment:
I'm not sure metadata are meant to be found each time (you still get a dataframe), but yes we could definitely change it to warning.

Reply:
Indeed, it should be a warning

Reply:
In fact,_add_metadata isn't used anymore in the project and has been "commented out" (transformed into a string to be precise) since [that commit](https://github.com/InseeFrLab/pynsee/commit/4b54d7d90da2a4610862c213faae9ead27323eb2#diff-d4927200c12ce2876c819c95d4d9d494d958a711c108cda8f0bce5d5e56384e0R20). @hadrilec should we remove it altogether ?

Reply:
Unless I have not understood this commit, I think the message is still relevant. @tgrandje could you please elaborate on why you think the message should be removed? Thanks

Reply:
I confirm that this function has been commented out: https://github.com/InseeFrLab/pynsee/blob/bf12359d859154abe015b6ca304ecaf1613f5919/pynsee/download/_get_value_label.py#L20

Reply:
@hadrilec I didn't comment it out, it's due to your previous PR (might be because the function is not called anymore in the project ?)